ui-backend/catalog-ui-search/pom.xml

```diff
@@ -158,19 +158,24 @@
         <dependency>
             <groupId>org.geotools</groupId>
             <artifactId>gt-opengis</artifactId>
-            <version>${opengis.version}</version>
+            <version>20.1</version>
```

Comment:
it's not clear why we wouldn't want to use the root level `opengis.version` here